src/repley/shadow/remote.cljs

```diff
@@ -1,22 +1,31 @@
 (ns repley.shadow.remote
   (:require [repley.config :as config]))
 
-(def default-port
-  (get-in config/default-config [:http :port]))
+(defn config->fetch-opts [{:keys [http] :as config}]
+  (cond-> config
+    (contains? http :port) (assoc :port (:port http))
+    true (select-keys [:port :prefix :receive-endpoint])))
+
+(comment
+  (config->fetch-opts {:http {:port 1} :receive-endpoint nil :prefix "/foo"}))
+
+(def default-fetch-opts
+  (-> config/default-config
+      config->fetch-opts
+      (assoc :mode "cors")))
 
 (defn make-send [fetch]
   (fn submit
     ([value] (submit nil value))
-    ([{:keys [port mode]
-       :or   {port default-port
-              mode "cors"}}
-      value]
-     (fetch (str "http://localhost:" port "/receive")
-            {:method  "POST"
-             :mode    mode
-             :headers {"content-type" "application/edn"}
-             :body    (binding [*print-meta* true]
-                        (pr-str value))}))))
+    ([opts value]
+     (let [{:keys [port mode prefix receive-endpoint]} (merge default-fetch-opts opts)]
+       (when (some? receive-endpoint) ;; nil disables the endpoint
```

Comment:
Note that `:receive-endpoint nil` doesn't actually disable the endpoint on the server.

Reply:
It should never match and thus won't be called if configured to `nil`

Reply:
I don't think that's correct. The `repley.config/config` will return the default endpoint and then `repley-handler` will match against `localhost:port/receive`.

Reply:
I think using a keyword like `:repley/disable-endpoint` would make this easier :)